packages/button/button.scss

```diff
@@ -94,10 +102,11 @@ $button-primary-background-color--inverted: $hvit;
 
     // ********* VARIANTS *********
 
+    &--action,
     &--primary,
     &--secondary {
-        border: solid $button-border-width $button-border-color;
-        border: solid $button-border-width var(--button-border-color);
+        border: solid $button-border-width;
+        border: solid $button-border-width;
```

Comment:
wat?

Reply:
Fargen er forskjellig på de forskjellige knappene 😊 Settes rett under.

Reply:
jaja, fargen er grei, men hvorfor skal du sette border width to ganger..

Reply:
Haha, d'oh! Det trengs jo ikke nei 😆